### PR TITLE
Update twinPicks-compound.jsonld

### PR DESCRIPTION
**Pull request by junjou**
This compounds will be the core compounds context used by TP for the majority of the requests

### CODE REVIEW

File: twinPicks/jsonld-contexts/twinPicks-compound.jsonld [hunk 1]
```diff
@@ -1,11 +1,17 @@
 {
   "@context": [
-    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat-compound.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/datacenter-equipments.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/c2jn-smart-city-compound.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/archive/digital-twins/jsonld-contexts/dcat.jsonld",
```
Comment (bobeal) [inline twinPicks/jsonld-contexts/twinPicks-compound.jsonld:6]: two `dcat.jsonld`?

File: twinPicks/jsonld-contexts/twinPicks-compound.jsonld [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
     "https://easy-global-market.github.io/ngsild-api-data-models/twinPicks/jsonld-contexts/usecase.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/device/jsonld-contexts/device.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/environment/jsonld-contexts/environment.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/weather/jsonld-contexts/weather.jsonld",
```
Comment (bobeal) [inline twinPicks/jsonld-contexts/twinPicks-compound.jsonld:10]: don't think they should go in the core TP context

File: twinPicks/jsonld-contexts/twinPicks-compound.jsonld [hunk 3]
```diff
@@ -1,11 +1,17 @@
 {
   "@context": [
-    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat-compound.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/datacenter-equipments.jsonld",
```
Comment (bobeal) [inline twinPicks/jsonld-contexts/twinPicks-compound.jsonld:3]: in the future, this one and `c2jn-smart-city-compound.jsonld` shoud be moved away from the core TP (but that may be difficult to do right now)

File: twinPicks/jsonld-contexts/twinPicks-compound.jsonld (hunk 2 above)
Comment (junjou) [reply twinPicks/jsonld-contexts/twinPicks-compound.jsonld:10]: I'm not sure about those... There is a big chance that they will be needed in all usecases.

File: twinPicks/jsonld-contexts/twinPicks-compound.jsonld (hunk 1 above)
Comment (junjou) [reply twinPicks/jsonld-contexts/twinPicks-compound.jsonld:6]: Don't sure which one to keep. But we should stop using the /archive anyways.

File: twinPicks/jsonld-contexts/twinPicks-compound.jsonld (hunk 2 above)
Comment (junjou) [reply twinPicks/jsonld-contexts/twinPicks-compound.jsonld]: Removed

**Review by bobeal — APPROVED** [review]

**Review by gasserluc — APPROVED** [review]

**Pull request merged**